sdk/ml/azure-ai-ml/azure/ai/ml/_schema/registry/system_created_storage_account.py

```diff
@@ -15,10 +15,13 @@
 @experimental
 class SystemCreatedStorageAccountSchema(metaclass=PatchedSchemaMeta):
     arm_resource_id = fields.Str(dump_only=True)
-    storage_account_hns = fields.Bool()
+    storage_account_hns = fields.Bool(load_default=False)
     storage_account_type = StringTransformedEnum(
+        load_default=StorageAccountType.STANDARD_LRS,
         allowed_values=[accountType.value for accountType in StorageAccountType], casing_transform=lambda x: x.lower()
     )
+    replication_count = fields.Int(load_default=1, validate=lambda count : count > 0)
```

Comment:
Minor: Usually entity is a better place to add validation, as SDK users can also get the validation.

Reply:
Added another check in the entity-API object conversion function, and checked for this in tests.